packages/icons/src/Feedback.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Svg } from './Svg';
+
+export const Feedback = () => (
+  <Svg>
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M80 12.5H20C15.875 12.5 12.5375 15.875 12.5375 20L12.5 87.5L27.5 72.5H80C84.125 72.5 87.5 69.125 87.5 65V20C87.5 15.875 84.125 12.5 80 12.5Z"
+      fill="black"
```

Comment:
Can not have a fun otherwise it will not be able to change to another color.

Same for the other attributes. Are they needed?